tools/pd-ctl/pdctl/command/gc_safepoint_command.go

```diff
@@ -15,16 +15,8 @@
 package command
 
 import (
-	"encoding/json"
-	"net/http"
-	"sort"
-
 	"github.com/spf13/cobra"
-	"github.com/tikv/pd/server/api"
-)
-
-var (
-	serviceGCSafepointPrefix = "pd/api/v1/gc/safepoint"
+	"sort"
```

Comment:
Please format this part. The import of the standard library should be placed before third-party imports and should include a blank line.